exporter/signalfxexporter/config.go

```diff
@@ -202,7 +202,7 @@ func (cfg *Config) Unmarshal(componentParser *confmap.Conf) error {
 		return nil
 	}
 
-	if err := componentParser.Unmarshal(cfg); err != nil {
+	if err := componentParser.Unmarshal(cfg, confmap.WithIgnoreUnused()); err != nil {
```

Comment:
Migrating to explicitly call out this unmarshaling ignores unused elements. The default behavior has changed to error out on unused elements.